EduVPN-multiOS/Coordinators/AppCoordinator+Repositories.swift

```diff
@@ -124,35 +128,14 @@ extension AppCoordinator {
                     throw error
                 }
                 
-                func retryFetchProfile() -> Promise<[String]> {
```

Comment:
Is there a swiftlint rule against inline functions? Why?

Reply:
This actually to get around a function length rule.

This is the output I resolved by these changes: `Function Body Length Violation: Function body should span 40 lines or less excluding comments and whitespace: currently spans 53 lines (function_body_length)`

Reply:
I do prefer to not use inline functions. Especially if capturing of context is not strictly required or very straight forward. A one or two line body, that's fine. ut anything more, I'd say better to factor that to a private function outside of the function body.

Reply:
I can live with this change. I do sometimes like to have a function inline so that it is clear that is only expected to be called from a limited scope.